packages/slice-machine/src/components/Card/Card.css.ts

```diff
@@ -0,0 +1,142 @@
+    },
+  },
+});
+
+export const interactiveVariant = {
+  solid: sprinkles({
+    backgroundColor: selectors.focusVisible(colors.grey5),
+    borderColor: {
+      ...selectors.checked(colors.purple10),
+      ...selectors.focusVisible(colors.purple10),
+    },
+  }),
+  outlined: sprinkles({
+    backgroundColor: {
+      ...selectors.active(colors.grey4),
+      ...selectors.focusVisible(colors.grey1),
+      ...selectors.hover(colors.grey4),
+    },
+    borderColor: {
+      ...selectors.active(colors.purple9),
+      ...selectors.checked(colors.purple8),
+      ...selectors.focusVisible(colors.purple8),
+    },
+  }),
+};
+
+export const media = style([
+  sprinkles({ boxSizing: "border-box" }),
+  {
+    order: 0,
+    selectors: {
+      [`${size.small} > &`]: { height: "180px" },
+      [`${size.medium} > &`]: { height: "260px" },
+      [`${variant.solid} > &`]: {
+        padding: calc.subtract(vars.space[32], vars.borderWidth[1]),
+      },
+      [`${variant.outlined} > &`]: {
+        backgroundColor: vars.color.greyLight3,
+        marginInline: calc.subtract(vars.space[16], vars.borderWidth[1]),
+        marginTop: calc.subtract(vars.space[16], vars.borderWidth[1]),
+        padding: vars.space[16],
+      },
+    },
+  },
+]);
+
+export const mediaComponent = styleVariants({
+  div: [sprinkles({ all: "unset", display: "grid" })],
+  img: [
+    sprinkles({ objectFit: "contain" }),
+    { selectors: { "&:not([src])": { visibility: "hidden" } } },
+  ],
+});
+
+const nonMedia = style([
+  flex,
+  sprinkles({
+    alignItems: "center",
+    boxSizing: "border-box",
+    flexDirection: "row",
+  }),
+  {
+    selectors: {
+      [`${variant.solid} > &`]: { backgroundColor: vars.color.greyLight1 },
+      [`${variant.solid} > &:not(:first-child)`]: {
+        borderTopColor: vars.color.greyLight6,
+        borderTopStyle: "inherit",
+        borderTopWidth: "inherit",
+      },
+    },
+  },
+]);
+
+export const actions = style([
+  nonMedia,
+  sprinkles({ flexWrap: "wrap", height: 48, justifyContent: "space-between" }),
+  {
+    order: 1,
+    selectors: {
+      [`${variant.solid} > &`]: {
+        paddingLeft: calc.subtract(vars.space[12], vars.borderWidth[1]),
+        paddingRight: calc.subtract(vars.space[8], vars.borderWidth[1]),
+      },
+      [`${variant.outlined} > &`]: {
+        paddingInline: calc.subtract(vars.space[16], vars.borderWidth[1]),
+      },
+    },
+  },
+]);
+
+export const footer = style([
+  nonMedia,
+  {
+    height: "60px",
+    order: 2,
+    selectors: {
+      [`${variant.solid} > &`]: {
+        paddingInline: calc.subtract(vars.space[12], vars.borderWidth[1]),
+      },
+      [`${variant.outlined} > &`]: {
+        paddingInline: calc.subtract(vars.space[16], vars.borderWidth[1]),
+      },
+    },
+  },
+]);
+
+export const footerTexts = style([
+  column,
+  sprinkles({ flexGrow: 1, minWidth: 0 }),
+  { marginBottom: `calc((${24 / 16}rem - ${16 / 16}rem) / 2)` },
```

Comment:
❓ Where does this come from?

Reply:
Yeah no problem with the editor problem. A small comment could be nice for maintenance yes 🙌